src/codeManager.ts

```diff
@@ -426,14 +427,19 @@ export class CodeManager implements vscode.Disposable {
         }
         this.sendRunEvent(executor, false);
         const startTime = new Date();
-        this._process = exec(command, { cwd: this._cwd });
+        this._process = exec(command, { cwd: this._cwd, encoding: "Buffer" });
 
-        this._process.stdout.on("data", (data) => {
-            this._outputChannel.append(data);
+        let encoding = this._config.get<string>("outputEncoding");
+        encoding = iconv.encodingExists(encoding) ? encoding : "utf8";

```

Comment:
Put this two line of code before `const startTime = new Date();`

Reply:
okay